docs/src/solutions.md

```diff
@@ -0,0 +1,74 @@
+Querying Solutions
+==================
+
+So far we have seen all the elements and constructs related to writing a JuMP
+optimization model, in this section we reach the point of what to do
```

Comment:
Comma splice detected.

Solution would be to make the sentences independent (period), make the sentences dependent (conjunction after comma to join sentences) or use a semicolon.